### PR TITLE
documentation & broken task updates

### PR DESCRIPTION
**Pull request by benjtinsley**
this is a catch all PR that fixes a few things and updates some wording in anticipation of a beta 5.0 release

### CODE REVIEW

File: gulpfile.js/tasks/init.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ var projectPath = require('../lib/projectPath')
 var merge       = require('merge-stream')
 
 gulp.task('init', function() {
-  var rootStream = gulp.src('root/*')
-    .pipe(gulp.dest(projectPath()))
+  var rootStream = gulp.src(['root/.eslintrc.js', 'root/.gitignore', 'root/.nvmrc', 'root/.prettierignore', 'root/.stylelintrc'])
```
Comment (benjtinsley) [inline gulpfile.js/tasks/init.js]: i couldn't get globbing to work with `.` prefixed files, so had to list them out explicitly. if there is a better way, please let me know

File: gulpfile.js/tasks/init-config.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+var merge = require('merge-stream')
+
+gulp.task('init-config', function () {
+  var configStream = gulp.src(['gulpfile.js/path-config.json', 'gulpfile.js/task-config.js'])
+    .pipe(gulp.dest(projectPath('config')))
+
+  log(colors.green('Adding default path-config.json and task-config.js files to ./config/'))
+
+  return configStream
+})
```
Comment (benjtinsley) [inline gulpfile.js/tasks/init-config.js]: the init-config task does need to stay in after all, as defined here: https://github.com/vigetlabs/blendid/wiki/Configuration

File: README.md [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
 
-**Blendid** *(formerly known as Gulp Starter)* is a delicious stand-alone blend of tasks and build tools poured into [Gulp](http://gulpjs.com/) to form a full-featured modern asset pipeline. It can be used as-is as a static site builder, or can be configured and integrated into your own development environment and site or app structure.
+**Blendid** is a delicious stand-alone blend of tasks and build tools that form a full-featured modern asset pipeline. It can be used as-is as a static site builder, or can be configured and integrated into your own development environment and site or app structure.
 
 ## Quick start on a fresh project (empty directory)
 ```bash
 yarn init
 yarn add blendid
-yarn run blendid -- init
+yarn run blendid init
```
Comment (benjtinsley) [inline README.md]: yarn no longer accepts `--` for tasks.

from the yarn console warning:

```
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
```

File: README.md [hunk 4]
```diff
@@ -3,13 +3,13 @@
 [![Build Status](https://travis-ci.org/vigetlabs/blendid.svg?branch=static-server)](https://travis-ci.org/vigetlabs/blendid)
 [![Stories in Ready](https://badge.waffle.io/vigetlabs/blendid.png?label=ready&title=Ready)](https://waffle.io/vigetlabs/blendid)
 
-**Blendid** *(formerly known as Gulp Starter)* is a delicious stand-alone blend of tasks and build tools poured into [Gulp](http://gulpjs.com/) to form a full-featured modern asset pipeline. It can be used as-is as a static site builder, or can be configured and integrated into your own development environment and site or app structure.
+**Blendid** is a delicious stand-alone blend of tasks and build tools that form a full-featured modern asset pipeline. It can be used as-is as a static site builder, or can be configured and integrated into your own development environment and site or app structure.
```
Comment (benjtinsley) [inline README.md]: i want to start moving away from gulp as the backbone of blendid and pivot towards more on a non-tech specific front-end asset pipeline tool that can have whatever as the backbone (webpack, parcel, etc) as long as its battle tested and can accomplish the tasks we need

File: gulpfile.js/tasks/init.js (hunk 1 above)
Comment (olets) [reply gulpfile.js/tasks/init.js]: Did you try `'root/.*'`? `*` won't match files hidden with an initial `.`. (Doesn't look like it's needed in this case, but `'root/{.,}*'` should match every direct child of `root/`)

**Comment by olets** [comment]
p.s. nice, I'm into this direction

**Pull request merged**